src/app/import/components/modal-upload/modal-upload.component.ts

```diff
@@ -0,0 +1,87 @@
+import { Component, ElementRef, EventEmitter, Input, OnChanges, OnInit, Output, SimpleChanges, ViewChild } from '@angular/core';
+import { createDefaultModalUploadModal, getSubtitle, getTitle, UploadModals } from './modal-upload-helper';
+import { UploadFileType, UploadOptions } from '../helpers/upload-types';
+import { UploadComponent } from '../upload/upload.component';
+import { UploadStates } from '../../services/s3.enums';
+import { interval } from 'rxjs';
+
+@Component({
+  selector: 'kern-modal-upload',
+  templateUrl: './modal-upload.component.html',
+  styleUrls: ['./modal-upload.component.scss']
+})
+export class ModalUploadComponent implements OnInit, OnChanges {
+
+  @Input() uploadFileType: UploadFileType;
+  @Input() isModalOpen: boolean;
+  @Input() uploadOptions: UploadOptions;
+  @Input() projectId: string;
+
+  @Output() closeModalEvent = new EventEmitter();
+  @Output() fileAttached = new EventEmitter<File>();
+  @Output() refetchLookupLists = new EventEmitter<boolean>();
```

Comment:
this is still an output event which I don't understand => shouldn't exist, the component using the modal-uploadl component should decide when it wants to refetch (on normal close event)